### ISSUE TITLE
reset button

### ISSUE THREAD

**Issue by dimovnike** [OnionIoT/Onion-Hardware#6]
Hello, is there a bug in Omega-Expansion-Dock.pdf on SW2 (reset button)? All 4 terminals appear connected to GND permanently together with VDD25 and RESET_BTN.

In Omega-Mini-Dock.pdf its not clear either.

How reset is supposed to be activated (the GPIO38)? Is it active low and pulled up to 3.3V?
Thanks.

**Comment by chmorgan** [comment]
@dimovnike agreed. The schematic symbol is confusing as all get out. GPIO38 reset, FW_RST, is active high. The MCU reset pin is active low.
